src/libs/actions/ReimbursementAccount/resetFreePlanBankAccount.ts

```diff
@@ -98,7 +97,7 @@ function resetFreePlanBankAccount(bankAccountID: number | undefined, session: On
                         [INPUT_IDS.BUSINESS_INFO_STEP.STATE]: '',
                         [INPUT_IDS.BUSINESS_INFO_STEP.ZIP_CODE]: '',
                         [INPUT_IDS.BUSINESS_INFO_STEP.COMPANY_PHONE]: '',
-                        [INPUT_IDS.BUSINESS_INFO_STEP.COMPANY_WEBSITE]: getDefaultCompanyWebsite(session, user),
+                        [INPUT_IDS.BUSINESS_INFO_STEP.COMPANY_WEBSITE]: undefined,
```

Comment:
The value reset here is for the form draft, so it does have no effect on `defaultCompanyWebsite`. If we use an empty string, then the default value won't be applied.
https://github.com/Expensify/App/blob/53eee53ac8bfbdfd4d8eeaf16940e3b165483373/src/components/Form/FormProvider.tsx#L252-L255

I updated the `defaultCompanyWebsite` from `??` to `||` because `reimbursementAccount?.achData?.website` is a string, so I think it's safer to use `||`. Do you think we should just revert that?

Reply:
Done

Reply:
NAB: Would null be better to remove the key from Onyx?